src/upgrade/steps/env-refresh.js

```
@@ -0,0 +1,100 @@
+  } catch {
+    return false;
+  }
+}
+
+async function plan() {
+  const files = listAppEnvFiles();
+  if (files.length === 0) return 'no app .env files found in autopg root';
+  return `would verify+rewrite ${files.length} env file(s): ${files.map((f) => path.basename(f)).join(', ')}`;
+}
+
+async function execute({ log, warn }) {
```

ESLint

[error] 'log' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)